scss/mixins/typo.scss

```
@@ -0,0 +1,112 @@
+@mixin fontSen {
+    font-family: "Sen", Arial, Helvetica, sans-serif;
+}
+
+@mixin fontInter {
+    font-family: "Inter", Arial, Helvetica, sans-serif;
+}
+
+@mixin headingDisplay($color: $darkBg) {
+    @include fontSen;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 5.6rem;
+    line-height: 6.4rem;
+    letter-spacing: -0.2rem;
+    color: $color;
```

Chỗ này em có thể tối ưu hơn nữa bằng cách viết một cái mixin fontStyle cho các thuộc tính của font rồi @include vào, nó sẽ gọn hơn rất nhiều, áp dụng tương tự những thằng bên dưới
```suggestion
    @include fontSen;
    @include fontStyle($fontStyle: normal, $fontWeight: 700, $fontSize: 5.6rem, $lineHeight6.4rem, $letterSpacing: 0.2rem, $textColor: $color)
```